app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Admin::DashboardController < ApplicationController
 
   def index
-
+    @customers = Customer.top_customers
```

Comment:
Maybe name this variable @top_customers so its clear in the view which customers are being listed?